test/common/packagelib.py

```diff
@@ -87,7 +87,9 @@ def setUp(self):
             self.machine.execute(f"mkdir -p {empty_repo_dir} || true")
             self.machine.execute(f"repo-add {empty_repo_dir}/empty.db.tar.gz")
             self.machine.write("/etc/pacman.conf", config)
-            self.machine.execute("pacman -Sy")
+            # Clean up possible leftover lockfile
+            self.machine.execute("test -f /var/lib/pacman/db.lck && fuser -k /var/lib/pacman/db.lck && sleep 0.2")
```

Comment:
This [fails all over the place](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18869-20230531-175624-6d647051-arch-bots-4818/log.html#199-2). Please avoid using `&&`, it has a really strange semantics. In particular `false && true` is false, which may cause this. Perhaps `if` or `while`?